gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/rolesprovider/UserRolesProvider.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.security.rolesprovider
+
+public interface UserRolesProvider {
+
+  /**
+   * Load the roles assigned to the {@link com.netflix.spinnaker.security.User User}.
+   *
+   * @param userEmail
+   * @return Roles assigned to the {@link com.netflix.spinnaker.security.User User} with the given userEmail.
+   */
+  public Collection<String> loadRoles(String userEmail)
```

Comment:
ok. Just to anticipate a little bit, do you know how roles will be used? Will there be a list of static roles or will it be a configurable mapping?